### DISCUSSION TITLE
Pictures for Presentations

### DISCUSSION THREAD

**Discussion by djangodemetri** [glazet/QuadEVCharger#14] [Show and tell]
![CP Operation Chart](https://github.com/glazet/QuadEVCharger/assets/131516746/6faeac16-cfe5-4e2a-a4d1-b520b9c52047)
![Duty Cycle Chart](https://github.com/glazet/QuadEVCharger/assets/131516746/7819cd59-2b8e-4a84-8d5d-0e2d6ea61515)
![4 Car Parking Layout](https://github.com/glazet/QuadEVCharger/assets/131516746/aa01dbc8-6025-4d33-aca1-f964fadb6954)

**Comment by glazet** [comment]
![image](https://github.com/glazet/QuadEVCharger/assets/145175717/f8053cc3-d07c-4dad-bbbb-5d45700dbb0c)
